src/object-cache.cls.php

```diff
@@ -133,6 +134,26 @@ public function __construct($cfg = false)
 			$this->_cfg_enabled = false;
 		}
 	}
+	
+	/**
+	 * Add debug.
+	 *
+	 * @since  6.3
+	 * @access private
+	 */
+	private function debug_function( $text, $error = false ){
+	    if( defined('LSCWP_LOG') ){
+	        Debug2::debug( $text );
+	    }
+	    else{
+	        // Show messages if WP_DEBUG is on.
+	        $show = defined('WP_DEBUG') ? WP_DEBUG : false;
+	        // Show if text is error.
+	        $error && $show = true;
```

Comment:
No reason to use this way, not good to read. 

Also, no need to use extra var, just reuse $error.

Reply:
Changed